assets/js/blocks/cart-checkout/cart-i2/edit.tsx

```diff
@@ -176,6 +176,13 @@ const BlockSettings = ( { attributes, setAttributes } ) => {
  * @param {function(any):any} props.setAttributes Setter for attributes.
  */
 const CartEditor = ( { className, attributes, setAttributes } ) => {
```

Comment:
Some easier TS errors you can solve, even though linting was disabled at the top.